common/src/main/scala/explore/cache/ProgramCache.scala

```diff
@@ -205,7 +217,17 @@ object ProgramCache extends CacheComponent[ProgramSummaries, ProgramCache]:
           .compose(ProgramSummaries.proposalAttachments.replace(proposalAttachments))
       ))
 
+    val updatePrograms =
+      ProgramQueriesGQL.ProgramInfoDelta
+        .subscribe[IO]()
+        .map(
+          _.map(data =>
+            ProgramSummaries.programs
+              .modify(_.updated(data.programEdit.value.id, data.programEdit.value))
+          )
+        )
+
     // TODO Handle errors, disable transparent resubscription upon connection loss.
-    (updateTargets, updateObservations, updateGroups, updateAttachments).mapN(
-      _.merge(_).merge(_).merge(_)
+    (updateTargets, updateObservations, updateGroups, updateAttachments, updatePrograms).mapN(
+      _.merge(_).merge(_).merge(_).merge(_)
```

Comment:
lots of merging

Reply:
Is there a better way?

Reply:
not sure what merge does, maybe `foldLeft`?

Reply:
> not sure what merge does, maybe `foldLeft`?

Raul thumbed up this comment. I'm not sure what that means.

Reply:
That we can use `List(...).reduceLeft(_ merge _)`.

Reply:
Ah. I change that.

Reply:
Actually, I think it was simpler with `sequence.map` in this case.